index.html

```diff
@@ -30,72 +30,108 @@ <h1>AMBROSE <br> ISIGBA</h1>
+
          <div id="list-of-project1" class="list-of-project">
           <h4>Project names goes here</h4>
           <ul>
                <li class="project-list-1">HTML/CSS</li>
                <li class="project-list-2">Ruby on Rails</li>
                <li class="project-list-3">Javascricpt</li>
+               <li class="project-list-4">+1</li>
+
            </ul>
            <div class="project-link"><a href="#">See this project</a>
             <img class="img-list" src="./images/arrow-right.png" alt="arrow"></div>
          </div>
        </div>     
        <div  class="full-section-div">
-        <div id="img-div2" class="imj-project-section" ><img id="full-section-div2" src="./images/desktop2.png" alt=""></div>
+        <div id="img-div2" class="imj-project-section" >
+          <img id="full-section-div2" src="./images/desktop2.png" alt="">
+          <img id="list-of-project2-img" src="./images/desktop5.png" alt="">
+          <img id="list-of-project2-imgb" src="./images/Group-83-copy.png" alt="">
+        </div>
+
         <div id="list-of-project2" class="list-of-project">
          <h4>Project names goes here </h4>
          <ul>
               <li class="project-list-1">HTML/CSS</li>
               <li class="project-list-2">Ruby on Rails</li>
               <li class="project-list-3">Javascricpt</li>
+
+              <li class="project-list-4">+1</li>
+
           </ul>
           <div class="project-link"><a href="#">See this project</a>
            <img class="img-list" src="./images/arrow-right.png" alt="arrow"></div>
         </div>
       </div>     
       <div class="full-section-div">
-        <div id="img-div3" class="imj-project-section"><img id="full-section-div3" src="./images/desktop3.png" alt=""></div>
+
+        <div id="img-div3" class="imj-project-section">
+          <img src="./images/image-geometry5c.png" alt="" id="desktop-imgA">
+          <img id="full-section-div3" src="./images/desktop3.png" alt="">
+          <img id="desktop-img3" src="./images/image-geometry_4-1.png" alt="">
+        
+        </div>
+
         <div id="list-of-project3" class="list-of-project">
          <h4>Project names goes here </h4>
          <ul>
               <li class="project-list-1">HTML/CSS</li>
               <li class="project-list-2">Ruby on Rails</li>
               <li class="project-list-3">Javascricpt</li>
+
+              <li class="project-list-4">+1</li>
+
           </ul>
           <div class="project-link"><a href="#">See this project</a>
            <img class="img-list" src="./images/arrow-right.png" alt="arrow"></div>
         </div>
       </div>     
       <div   class="full-section-div">
-        <div class="imj-project-section"><img id="full-section-div4" src="./images/Desktop4.png" alt=""></div>
+
+        <div class="imj-project-section">
+          <img id="destop-img4" src="./images/Shape1b.png" alt="">
+          <img id="full-section-div4" src="./images/Desktop4.png" alt=""></div>
+
```

Comment:
- [ ] Your fourth project is being cut out of the screen in some screen sizes. Kindly fix that.
- [ ] The appropriate size of your third project is around 50% of the screen and the fourth project takes about 30% and starts in the middle of the screen. Kindly try to follow the Figma template for positioning and sizing your project's images.
Your project:
![image](https://user-images.githubusercontent.com/61919617/172938267-5c7b483c-6250-4dfb-9453-a29f12548667.png)
Figma template:
![image](https://user-images.githubusercontent.com/61919617/172938347-36b99670-0336-443f-a48a-cd51eaf8065b.png)